observations/obs_converters/ocean_color/work/mkmf_advance_time

```diff
@@ -0,0 +1,18 @@
+#!/bin/csh
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+#
+# DART $Id$
```

Comment:
since these are new files, suggest removing old SVN lines before their first commit -- anything with $Id$ and last 4 lines of file w/ $URL$ etc.

Reply:
leaving these as is.